package.json

```diff
@@ -27,6 +29,7 @@
     "start:prod": "yarn assets && yarn build:server && NODE_ENV=production yarn start",
     "start": "scripts/start.sh",
     "sync-schema": "rm -rf data && mkdir data && graphql-fetch-schema https://metaphysics-staging.artsy.net -o data",
+    "test:smoke": "scripts/smoke_test.sh",
```

Comment:
same here - maybe rename script to `cypress` or `run_cypress` as well.

Reply:
@eessex - thoughts on taking over old `acceptance` name here as well?